src/components/MessagesList/MessagesList.vue

```diff
@@ -284,6 +284,7 @@ export default {
 		EventBus.$on('scroll-chat-to-bottom-if-sticky', this.scrollToBottomIfSticky)
 		EventBus.$on('focus-message', this.focusMessage)
 		EventBus.$on('route-change', this.onRouteChange)
+		EventBus.$on('message-edited', this.handleMessageEdited)
```

Comment:
Could also watch after store getter instead of EventBus here

Reply:
It might be tricky to watch over a store getter which is probably `messageIdToEdit` because the id can also be removed upon abort too or start a reply message. For the sake of clarity, let's keep the event :3 .

Reply:
Or just discard an editing process, because replying is a normal messaging, so you're basically switching from editing. I'm not sure someone is expecting to continue to edit message after sending another one.

Reply:
It makes sense for me now, beacuse we also need to look for external edits